tools/post-process-alex-tables.hs

```diff
@@ -78,3 +79,8 @@ process strings = loop "" $ addImports strings
 
   convert :: String -> String
   convert s = "  $ decode $ fromString " ++ show (encode (read s :: [Int]))
+    where
+    decoded :: [Int]
```

Comment:
It doesn't look like you're using `decoded` anywhere - is there some code you forgot to commit?